### PR TITLE
Redesign page

### PR DESCRIPTION
**Pull request by ghost**
## Summary

- Redisign of the page

## Screenshots

Captura de pantalla de 2019-05-27 09-53-42

## Trello Card

https://trello.com/c/s1OCOUU6/12-redise%C3%B1o-de-la-p%C3%A1gina

### CODE REVIEW

File: maquetado/index.html [hunk 1]
```diff
@@ -17,31 +17,31 @@ <h2 class="navbar-text">Developer Tools</h2>
...
     <div class="wrap">
-      <div class="technology">
+      <div class="technology first-card">
         <img class="logo" src="./assets/react_icon.svg">
         <h4 class="technology-title">ReactJS</h4>
         <h4 class="technology-release">Release:2013</h4>
         <p class="technology-description">React is a declarative, component-based JavaScript library for building user interfaces.</p>
       </div>
-      <div class="technology">
+      <div class="technology cards">
```
Comment (mmamaniwolox) [inline maquetado/index.html]: Instead of adding style to every tech, why do you think of changing the `technology` class and just override in the case of `first-card`?

**Review by mmamaniwolox — CHANGES_REQUESTED** [review]

File: maquetado/index.html [hunk 2]
```diff
@@ -11,76 +11,76 @@
 </head>
 <body>
   <div class="navbar">
-    <img class="navbar-logo" src="./assets/wolox_logo.svg" alt="wolox">
+    <img class="navbar-logo" src="./assets/wolox_logo.svg">
```
Comment (mmamaniwolox) [inline maquetado/index.html]: why did you remove the alt?

File: maquetado/index.html [hunk 3]
```diff
@@ -11,76 +11,76 @@
...
     <h2 class="navbar-text">Developer Tools</h2>
   </div>
   <h2 class="title">Front-End Tools</h2>
-  <div class="container">
+  <div class="technologies">
     <div class="wrap">
-      <div class="technology">
-        <img class="logo" src="./assets/react_icon.svg" alt="React">
+      <div class="technology first-card">
+        <img class="logo" src="./assets/react_icon.svg">
```
Comment (mmamaniwolox) [inline maquetado/index.html]: close img tag

**Review by mmamaniwolox — CHANGES_REQUESTED** [review]

**Comment by Dorian30** [comment]
You have some conflicts here

**Review by mmamaniwolox — APPROVED** [review]

**Review by Dorian30 — APPROVED** [review]
